service/service.go

```diff
@@ -5,20 +5,17 @@ import (
 	"fmt"
 	"sync"
 
-	appv1alpha1 "github.com/giantswarm/apiextensions/v2/pkg/apis/application/v1alpha1"
-	infrastructurev1alpha2 "github.com/giantswarm/apiextensions/v2/pkg/apis/infrastructure/v1alpha2"
-	releasev1alpha1 "github.com/giantswarm/apiextensions/v2/pkg/apis/release/v1alpha1"
-	corev1alpha1 "github.com/giantswarm/apiextensions/v3/pkg/apis/core/v1alpha1"
-	"github.com/giantswarm/k8sclient/v5/pkg/k8sclient"
-	"github.com/giantswarm/k8sclient/v5/pkg/k8srestconfig"
+	appv1alpha1 "github.com/giantswarm/apiextensions-application/api/v1alpha1"
+	corev1alpha1 "github.com/giantswarm/config-controller/api/v1alpha1"
```

Comment:
Is this correct? Looks weird to me, maybe I am just not reading the deps correctly.

Reply:
Yes. This is importing the `Config` type from the `core.giantswarm.io` group. It has been moved into `config-controller`. The rest of the group is still in `apiextensions`.